internal/api/grpc/user/v2/query_integration_test.go

```diff
@@ -19,6 +20,15 @@ import (
 	user "github.com/zitadel/zitadel/pkg/grpc/user/v2beta"
 )
 
+func setOrgID(ctx context.Context, orgID string) context.Context {
+	md, ok := metadata.FromOutgoingContext(ctx)
+	if !ok {
+		return metadata.AppendToOutgoingContext(ctx, "x-zitadel-orgid", orgID)
+	}
+	md.Set("x-zitadel-orgid", orgID)
+	return metadata.NewOutgoingContext(ctx, md)
+}
```

Comment:
please remove, this is v2 service and must not rely on this header

Reply:
removed that, used the tester function to change the password as it was a dependency function

Reply:
We need to merge https://github.com/zitadel/zitadel/pull/7641 first, as otherwise the context is still used in the change password for the resourceowner